internal/slackclient/client.go

```diff
@@ -162,6 +178,64 @@ func (c *SlackClient) get(path string, params map[string]string) ([]byte, error)
 	return body, nil
 }
 
+func (c *SlackClient) post(path string, params map[string]string, msg *WriteMessage) ([]byte, error) {
```

Comment:
This `post` method and `get` method can be abstracted such that we can create other such methods like `put` but I couldn't think of a good way to do this - so a little copying and pasting did the trick 😄